ESMF/CMakeLists.txt

```diff
@@ -0,0 +1,2 @@
+add_subdirectory(MAPL)
```

Comment:
Should this be "MAPL@" instead of "MAPL"

Reply:
Even more confused - I now found the yaml file and it has "@MAPL".  So this should not even work as it stands.

Reply:
Tom bad.   Did not realize that this was a new file when I was reading down.